Week-10/src/Components/Home.js

```diff
@@ -7,21 +7,20 @@ import { CopyToClipboard } from "react-copy-to-clipboard";
 import axios from "axios";
 
 const Home = () => {
-  const [value, setValue] = useState("");
-  const [input, setInput] = useState("");
+  const [getValue, setGetValue] = useState(""); 

```

Comment:
It should be value only. It's a variable not a function. Get Value means it's trying to get some value which makes it a function